.travis.yml

```diff
@@ -7,24 +7,16 @@ cache:
   directories:
     - node_modules
 
-branches:  # pushes only get built for master
-  only:
-    - master
```

Comment:
Overall this PR LGTM, but I'd keep the `only: master` build - I can't think of a reason why we would be interested in push-only builds for this repo. Not a huge deal, but it doubles the chance of PR build failures because there are two builds instead of one.